packages/lpa-questionnaire-web-app/src/lib/appeal-reply-api-wrapper.js

```diff
@@ -23,27 +25,19 @@ async function handler(path, method = 'GET', opts = {}, headers = {}) {
         const apiResponse = await fetch(url, {
           method,
           headers: {
-            'Content-Type': 'application/json',
             'X-Correlation-ID': correlationId,
             ...headers,
+            'Content-Type': 'application/json',
           },
           ...opts,
         });
 
         if (!apiResponse.ok) {
-          logger.debug(apiResponse, 'API Response not OK');
-          try {
-            const errorResponse = await apiResponse.json();
-            /* istanbul ignore else */
-            if (errorResponse.errors && errorResponse.errors.length) {
```

Comment:
3 weeks ago; damned if i can remember -i think i was leaning into 'not using the throwing of errors as a control mechanism' - if it isn't an actual error you shouldn't be needing to try/catch..  equally it might be the only thing i could figure out to unit-test it.